styles/styles.css

```diff
@@ -1,58 +1,88 @@
-@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&family=Russo+One&display=swap");
+@import url(./fonts.css);
 
 * {
-    font-family: "Roboto", sans-serif;
-    box-sizing: border-box;
+  font-family: "Roboto", sans-serif;
+  font-weight: 400;
+  box-sizing: border-box;
+  margin: 0;
 }
 
 .container {
-    background-color: #ffffff;
-    max-width: 1440px;
-    height: 570px;
+  background-color: #ffffff;
+  max-width: 1440px;
+  height: 570px;
 }
 
 .offer {
-    display: flex;
-    justify-content: center;
-    flex-direction: column;
-    height: 189px;
-    padding-top: 120px;
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  height: 189px;
+  padding-top: 120px;
 }
 
 .offer-title {
-    font-weight: 500;
-    font-size: 40px;
-    height: 47px;
-    margin: 0 auto;
-    width: 306px;
+  font-weight: 500;
+  font-size: 40px;
+  height: 47px;
+  margin: 0 auto;
+  width: 306px;
 }
 
 .offer-line {
-    height: 4px;
-    width: 100px;
-    margin: 18px auto 0;
-    background-color: #f5bd41;
+  height: 4px;
+  width: 100px;
+  margin: 18px auto 0;
+  background-color: #f5bd41;
 }
 
 .advantages {
-    display: flex;
-    margin: 0 auto;
-    justify-content: space-between;
-    height: 254px;
-    width: 1120px;
+  display: flex;
+  margin: 0 auto;
+  justify-content: space-between;
+  height: 254px;
+  width: 1120px;
 }
 
 .advantages-card {
-    display: flex;
-    flex-direction: column;
-    align-items: center;
-    justify-content: space-between;
-    margin-top: 84px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-between;
+  margin-top: 84px;
 }
 
```

Comment:
you need to add class for images and add with/height parameters - it`s important rule for images onpage